src/Maker/MakeTwigExtension.php

```diff
@@ -49,8 +50,14 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
 
     public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
     {
+        $name = $input->getArgument('name');
+
+        if (str_ends_with($name, 'Extension')) {
+            $name = str_replace('Extension', '', $name);
+        }
```

Comment:
Is this part needed? I thought the `createClassNameDetails` below would help append `Extension` but without duplicating it (e.g. `ExtensionExtension`)